public/index.html

```diff
@@ -5,17 +5,17 @@
 </head>
 <body>
   <div id="photo-gallery"></div>
-  <div id="checkout"></div>
+  <!-- <div id="checkout"></div> -->
   <div id="reviews"></div>
   <div id="recommendations"></div>
   <div id="experiences"></div>
   <script src="https://unpkg.com/react@16/umd/react.development.js" crossorigin></script>
   <script src="https://unpkg.com/react-dom@16/umd/react-dom.development.js" crossorigin></script>
   <script src="//unpkg.com/styled-components/dist/styled-components.min.js"></script>
-  <script src="http://localhost:3001/:listingid/bundle.js"></script>
-  <!-- <script src="http://localhost:3002/bundle.js"></script> -->
-  <script src="http://localhost:3003/bundle.js"></script>
-  <script src="http://localhost:3004/bundle.js"></script>
-  <script src="http://localhost:3005/bundle.js"></script>
+  <script src="http://3.133.19.147/:listingid/bundle.js"></script>
+  <!-- <script src="http://3.17.59.236/bundle.js"></script> -->
+  <script src="http://18.219.230.255/bundle.js"></script>
+  <script src="http://18.218.105.195:3004/bundle.js"></script>
+  <script src="http://18.191.134.91/bundle.js"></script>
```

Comment:
I used this link and my calendar wasnt working, maybe it would work for you?
"http://ec2-3-17-59-236.us-east-2.compute.amazonaws.com:3002/bundle.js"

Reply:
Hm, going to `http://3.17.59.236/bundle.js` works for me, but I'll play around with this - thanks!